addons/survey/views/survey_survey_views.xml

```diff
@@ -83,68 +84,68 @@
+                                <group string="Time &amp; Scoring" name="scoring">
+                                    <div colspan="2" class="o_checkbox_optional_field">
+                                        <label for="is_time_limited" string="Survey Time Limit"/>
+                                        <field name="is_time_limited" nolabel="1"/>
+                                        <div attrs="{'invisible': [('is_time_limited', '=', False)]}">
+                                            <field name="time_limit" widget="float_time" nolabel="1" class="oe_inline"/> minutes
+                                        </div>
+                                    </div>  
                                     <field name="scoring_type" widget="radio" />
                                     <field name="scoring_success_min" attrs="{'invisible': [('scoring_type', '=', 'no_scoring')]}" />
-                                    <field name="certification" attrs="{'invisible': [('scoring_type', '=', 'no_scoring')]}" />
-                                    <field name="certification_mail_template_id" attrs="{'invisible': [('certification', '=', False)]}" />
-                                    <label for="certification_report_layout" string="Certification Template" attrs="{'invisible': [('certification', '=', False)]}"/>
-                                    <div attrs="{'invisible': [('certification', '=', False)]}" class="d-flex">
-                                        <field name="certification_report_layout" class="oe_inline"/>
-                                        <button name="action_survey_preview_certification_template"
-                                            string="Preview" type="object"
-                                            icon="fa-external-link"  target="_blank" class="btn-link pt-0"/>
+                                    <div colspan="2" class="o_checkbox_optional_field">
+                                        <label for="certification"/>
+                                        <field name="certification" nolabel="1"/>
+                                        <div attrs="{'invisible': [('certification', '=', False)]}">
+                                            <field name="certification_report_layout" class="oe_inline"/>
+                                            <button name="action_survey_preview_certification_template"
+                                                string="Preview" type="object"
+                                                icon="fa-external-link"  target="_blank" class="btn-link pt-0"/>
+                                        </div>
+                                    </div> 
+                                    <field name="certification_mail_template_id"/>    
+                                    <div colspan="2" class="o_checkbox_optional_field" attrs="{'invisible': ['|', ('certification', '=', False), ('users_login_required', '=', False)]}">
```

Comment:
I think this messes a bit the layout:

![image](https://user-images.githubusercontent.com/41282536/141968893-d61dc242-3407-440b-97e2-5397fc05bc62.png)

See the vertical bar is not aligned correctly.
Can you double-check?

Reply:
I don't seem to have that problem. Do you still have it?
Here's what I got
![image](https://user-images.githubusercontent.com/82512965/142873404-a1e52197-033c-421a-9ed7-2f5966bbb5ad.png)